.devcontainer/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+# Debian and Ubuntu based images are supported. Alpine images are not yet supported.
```

Comment:
Nit: Could you please add a new line between the license header and the command about supported distros?